Cubical/Core/Prelude.agda

```diff
@@ -78,12 +78,58 @@ compPathP : {A : I → Set ℓ} → {x : A i0} → {y : A i1} → {B_i1 : Set 
   (p : PathP A x y) → (q : PathP (λ i → B i) y z) → PathP (λ j → ((λ i → A i) ∙ B) j) x z
 compPathP p q j = compPathP-filler p q j i1
 
+∙-rInv : (p : x ≡ y) → p ∙ (sym p) ≡ refl
```

Comment:
This proof and the next are in https://github.com/agda/cubical/blob/master/Cubical/Foundations/GroupoidLaws.agda